db/error_handler_test.cc

```diff
@@ -523,6 +523,8 @@ TEST_F(DBErrorHandlingTest, MultiDBCompactionError) {
   for (auto i = 0; i < kNumDbInstances; ++i) {
     std::string prop;
     ASSERT_EQ(listener[i]->WaitForRecovery(5000000), true);
+    ASSERT_EQ(static_cast<DBImpl*>(db[i])->TEST_WaitForCompact(true),
```

Comment:
Nit: you can also use `ASSERT_OK(static_cast<DBImpl*>(db[i])->TEST_WaitForCompact(true))`. Since `ASSERT_EQ(Status::OK(), ...)` is used elsewhere in this test, it is OK to keep this.

Reply:
I see.